trunk/SUAVE/Analyses/Aerodynamics/Vortex_Lattice.py

```diff
@@ -366,7 +363,7 @@ def sample_training(self):
         
         
         # Assign placeholders        
-        CL_sub    = np.zeros((len(AoA),len(Mach_sub)))
+        CL_sub    = np.zeros((len(AoA),len(Mach[0:8,0])))
```

Comment:
this indexing

Reply:
added the datalen syntax and removed the indexing